src/assets/date-picker.svg

```diff
@@ -0,0 +1,37 @@
+<svg width="106" height="97" viewBox="0 0 106 97" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Would it be better if we reuse the SVGs, we have in the website?

Reply:
These are the SVGs from the website, but this will avoid any case where the files are moved and break this overview list.